src/Proj4.jl

```diff
@@ -136,6 +142,9 @@ function transform!(src::Projection, dest::Projection, position::Array{Float64,2
     position
 end
 
+transform!(src::Projection, dest::Projection, position::Vector{Float64}; radians::Bool=false) =
+    transform!(src, dest, reshape(position,(1,length(position))), radians=radians)
+
```

Comment:
Nested vectors? Ick.

At work we deal with a lot of point clouds (not polygons), so the data is usually nicely structured.